recipe/meta.yaml

```
@@ -187,9 +187,9 @@ outputs:
 {% endif %}
     test:
       downstreams:
-        # Temporarily disable dowmstream tests on aarch64 as the solver causes timeouts on drone
-        - cython  # [not aarch64]
-        - setuptools  # [not aarch64]
+        # Temporarily disable downstream test for setuptools due to breaking changes
+        - cython
```

Guessing we still want the `not aarch64` selector due to the previous issue

```suggestion
        - cython  # [not aarch64]
```

---

Well, the comment was referencing drone, which we're not using anymore. Might as well try if it still times out. :)

---

Sure if you'd like to spend time trying/fixing it, that would of course be welcome :)

Though completely understand if you would rather stick to just getting the missing builds out

---

It passed on travis, so that should be good enough to keep this bit. ;-)